templates/stream.app.navigation.php

```diff
@@ -22,7 +22,7 @@
  *
  */
 ?>
-<div id="app-navigation">
+<div id="app-navigation" role="navigation">
 	<ul>
 	<?php foreach ($_['navigations'] as $navigation) { ?>
 		<li<?php if ($_['activeNavigation'] === $navigation['id']): ?> class="active"<?php endif; ?>>
```

Comment:
Also add `aria-current=page` on the initial loaded active navigation?

```suggestion
		<li<?php if ($_['activeNavigation'] === $navigation['id']): ?> class="active" aria-current="page"<?php endif; ?>>
```